conjureup/controllers/newcloud/tui.py

```diff
@@ -25,9 +25,8 @@ def render(self):
         # a user to configure a LXD bridge with suggested network
         # information.
         if cloud_type == 'localhost':
-            lxd = common.is_lxd_ready()
-            if not lxd['ready']:
-                return controllers.use('lxdsetup').render(lxd['msg'])
+            if common.is_lxd_ready():
+                return controllers.use('lxdsetup').render()
```

Comment:
Same as above, and the fact that this is repeated means it would be better to refactor into a `common.py` so we don't accidentally change something in one spot but not the other.